packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -223,6 +223,11 @@ async function handleExistingUsersInvites({
         parentId: team.parentId,
         accepted: false,
       });
+      await Promise.all(
+        regularUsers.map(async (user) => {
+          await updateNewTeamMemberEventTypes(user.id, team.id);
+        })
+      );
```

Comment:
Here regularUsers haven't accepted team invite so why are we assigning them to event types?